controller/src/main/java/io/pravega/controller/store/stream/PravegaTablesStoreHelper.java

```diff
@@ -176,7 +178,7 @@ public void invalidateCache(String table, String key) {
                 new TableEntryImpl<>(new TableKeyImpl<>(key.getBytes(Charsets.UTF_8), KeyVersion.NOT_EXISTS), value));
         Supplier<String> errorMessage = () -> String.format("addNewEntry: key: %s table: %s", key, tableName);
         return withRetries(() -> segmentHelper.updateTableEntries(tableName, entries, authToken.get(), RequestTag.NON_EXISTENT_ID),
-                errorMessage)
+                errorMessage, true)
```

Comment:
This change causes a slight change in the behavior of the controller wrt to createScope().

```
This change prevents retries in the case of `ConnectionDropped` or `ConnectionFailed`. In the case of `createScope()` 
workflow, this will prevent retries by the controller. And in this scenario the retries of the client would used.
```

Reply:
Yes, that is correct. But its not necessarily a behaviour change (except the bug fix). Its just fail faster rather than retry. 
Most updates happen from workflows. 
Some happen in the context of client calls.